<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration</class>
 <widget class="QMainWindow" name="registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>511</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>488</width>
    <height>511</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>488</width>
    <height>511</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AralaryShop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/rec/images/back_button.png</normaloff>:/rec/images/back_button.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="7" column="1">
                <widget class="QDateEdit" name="birth_line">
                 <property name="wrapping">
                  <bool>false</bool>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                 <property name="accelerated">
                  <bool>false</bool>
                 </property>
                 <property name="correctionMode">
                  <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                 </property>
                 <property name="maximumDate">
                  <date>
                   <year>2016</year>
                   <month>1</month>
                   <day>1</day>
                  </date>
                 </property>
                 <property name="minimumDate">
                  <date>
                   <year>1910</year>
                   <month>9</month>
                   <day>14</day>
                  </date>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                 <property name="date">
                  <date>
                   <year>2000</year>
                   <month>1</month>
                   <day>1</day>
                  </date>
                 </property>
                </widget>
               </item>
               <item row="11" column="1">
                <widget class="QLineEdit" name="email_line">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="18" column="2">
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="minimumSize">
                  <size>
                   <width>87</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>87</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Show</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="sname_line">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="18" column="1">
                <widget class="QLineEdit" name="pass2">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item row="11" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Email</string>
                 </property>
                </widget>
               </item>
               <item row="16" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item row="15" column="1">
                <widget class="QLabel" name="login_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="16" column="1">
                <widget class="QLineEdit" name="pass1">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item row="14" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Login</string>
                 </property>
                </widget>
               </item>
               <item row="12" column="1">
                <widget class="QLabel" name="mail_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Date of birth(dd.mm.yyyy)</string>
                 </property>
                </widget>
               </item>
               <item row="14" column="1">
                <widget class="QLineEdit" name="log_line">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label1">
                 <property name="text">
                  <string>First name</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="fname_line">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item row="19" column="1">
                <widget class="QLabel" name="pass_message">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Second name</string>
                 </property>
                </widget>
               </item>
               <item row="18" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Сonfirm the password</string>
                 </property>
                </widget>
               </item>
               <item row="16" column="2">
                <widget class="QCheckBox" name="checkBox1">
                 <property name="text">
                  <string>Show</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLabel" name="name_message">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>80</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Register</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>80</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>fname_line</tabstop>
  <tabstop>sname_line</tabstop>
  <tabstop>birth_line</tabstop>
  <tabstop>email_line</tabstop>
  <tabstop>log_line</tabstop>
  <tabstop>pass1</tabstop>
  <tabstop>pass2</tabstop>
  <tabstop>checkBox1</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
